runtime/bcutil/ROMClassBuilder.cpp

```diff
@@ -270,13 +270,32 @@ ROMClassBuilder::handleAnonClassName(J9CfrClassFile *classfile, bool *isLambda,
 	U_32 i = 0;
 	BOOLEAN stringOrNASReferenceToClassName = FALSE;
 	BOOLEAN newCPEntry = TRUE;
+	bool isLambdaForm = false;
 	BuildResult result = OK;
 	char buf[ROM_ADDRESS_LENGTH + 1] = {0};
 	U_8 *hostPackageName = context->hostPackageName();
 	UDATA hostPackageLength = context->hostPackageLength();
 	PORT_ACCESS_FROM_PORT(_portLibrary);
 
-#if defined(J9VM_OPT_OPENJDK_METHODHANDLE) && (JAVA_SPEC_VERSION >= 15)
+#if defined(J9VM_OPT_OPENJDK_METHODHANDLE)
```

Comment:
I see the following change is only for OpenJDK MH. For LambdaForm classes from MethodHandles, are they only generated by OpenJDK MH or they are also generated by OpenJ9 MH ?

Reply:
LambdaForms classes are only generated for OJDK MHs. With OJ9 MHs, the LambdaForm class is just an empty stub.